modules/3d-tiles/src/tiles-3d-loader.js

```diff
@@ -0,0 +1,135 @@
+/* global TextDecoder,  __VERSION__ */ // __VERSION__ is injected by babel-plugin-version-inline
+
```

Comment:
Assuming this and below new files are also moved/renamed, if not comment what sections are new so can be reviewed in detail.